CONTRIBUTING.md

```diff
@@ -139,6 +143,52 @@ The description field should
 - be neutral and not use biased additions like ~~leading~~, ~~innovative~~, ~~flexible~~ or buzzwords (~~performance~~, ~~next generation~~)
 - not contain hyperlinks or markup
 
+
+### Compatibility information
+
+FMU exporting and importing tools provide information to their users, how they have validated their FMI support.
+Providing and maintaining this information is in the responsiblity of the tool vendors; in the tools page of the FMI Webpage a link to a repository or webpage of the tool vendor will be provided.
+
+#### Exporting tools
+
+* List the importing tools (including version information), with which the FMUs from this exporter have been  imported and simulated successfully.
```

Comment:
This should be clarified that the exporter is not expected to purchase importing tools for the purposes of self certification. Is testing with just FMPy sufficient? If not, how do we incentivize importers to check FMUs exported from our tool?  We should outline a process for how exporters should reach out to importers and request this check.

Reply:
> This should be clarified that the exporter is not expected to purchase importing tools for the purposes of self certification. 

I added a sentence, please check. 

 > Is testing with just FMPy sufficient? 

In principle yes (if we do not agree to define stricter rules). But as an end user, I would ask such a tool vendor, why his exported FMUs have not been tested with other tools.

>  If not, how do we incentivize importers to check FMUs exported from our tool? 

They can report on their "compatibility information webpage" that they have tested the import and simulation of many tools.

>We should outline a process for how exporters should reach out to importers and request this check.

That is the disadvantage of not having a central cross-check repo any more, that one could just "watch" on Github.
Any idea how this could work?

Reply:
Thank you for the changes.
One idea for facilitating self-certification of FMUs is through the messaging app Signal.  Since we're going to use it for communications anyway, perhaps we can set up an FMI Cross Check channel where exporters can announce the availability of their FMUs (with a link to a webpage where they can be found) and invite import tool vendors to test them out.

Reply:
> One idea for facilitating self-certification of FMUs is through the messaging app Signal. Since we're going to use it for communications anyway, perhaps we can set up an FMI Cross Check channel where exporters can announce the availability of their FMUs (with a link to a webpage where they can be found) and invite import tool vendors to test them out.

The current proposal is for providing _compatibility information_ by the tools. There will be stricter rules for_self-certification_. The messenger Signal is currently used for informal communication within the FMI project, and should not be used be beyond (for privacy reasons)
The compatibility information and a possible notification process, when new uploads are available shall be usable beyond the FMI project, as we have many more tools on the tools list.  As long as we had a central cross-check repo, the Github machinery with watching and notifications was good for that purpose.  So parhaps we should move at least the tools list to a separate repo, and use the Github machinery for notfications of changes. 
@t-sommer : What do you think about this?

Reply:
FMI Design Meeting: 
Andreas J: we should clearly mark that this information is provided by the tool vendor and not by MA or the FMI Project.
Encourage to use your logos and mark this as PR.